src/material-experimental/mdc-list/interactive-list-base.ts

```diff
@@ -84,6 +85,7 @@ export abstract class MatInteractiveListBase<T extends MatListItemBase>
   protected _initWithAdapter(adapter: MDCListAdapter) {
     this._adapter = adapter;
     this._foundation = new MDCListFoundation(adapter);
+    this._foundation.setDisabledItemsFocusable(false);
```

Comment:
Does this have accessibility implications? I was thinking as per W3C, and matching with MDC, disabled items should remain focusable so that screenreaders can read them? although you could argue the other way around as well. Just checking to ensure we chat about it. https://www.w3.org/TR/wai-aria-practices-1.1/#kbd_disabled_controls

Reply:
@jelbourn what do you think about this? we mark these elements as `aria-disabled` so it felt weird to me to have them be focusable

Reply:
@devversion Hmm yeah reading through the document you linked it does sound like this is a specific exception case to the normal rule of not focusing disabled elements. I'll go ahead and change that